fossunited/integrations/github.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
             access_token = github_settings.get_password("access_token")
             self.github = Github(access_token)
         except Exception as e:
-            frappe.log_error(frappe.get_traceback(), _("Github Settings not found"))
```

Comment:
it's not clear to me what `_` is supposed to mean here or what it is supposed to do.

Reply:
If I am not wrong, it's used in translation mechanism - adding `_()` makes the text translatable to other languages, if there are browser workflows that support it. 

Otherwise, it brings in the original string itself

Reply:
thanks for the clarification. i'm happy to add it back but I don't see it's use in our context

Reply:
@harshtandiya can comment on this better, as to why it's used here. 

But in general sense, it's something that improves accessibility as well.

Reply:
It works either way imo

Reply:
thank you. not reverting the change and merging as-is. I've also resolved the other comment regarding the same